### PR TITLE
Remove errors while saving to favourites

### LINKED ISSUE #346: Error: When adding wallpaper as favourite
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.5/tkinter/__init__.py", line 1553, in __call__
    return self.func(*args)
  File "/usr/local/lib/python3.5/dist-packages/UIP-0.0.3-py3.5.egg/uiplib/gui/mainGui.py", line 133, in favourite
    copy(self.images[self.index], self.settings['fav-pics-folder'])
  File "/usr/lib/python3.5/shutil.py", line 234, in copy
    dst = os.path.join(dst, os.path.basename(src))
  File "/usr/lib/python3.5/posixpath.py", line 139, in basename
    i = p.rfind(sep) + 1
AttributeError: 'UipImage' object has no attribute 'rfind'
```

### PR DESCRIPTION
**Pull request by SACHIN-13**
fixes #346 
Also added a message box to intimate the user if the pic is added to favourites!
screenshot from 2017-01-31 10-43-31

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```
Also please upload a screenshot if you have made changes to the gui.

**Comment by abhsag24** [comment]
awesome!

File: uiplib/gui/mainGui.py [hunk 1]
```diff
@@ -130,4 +130,8 @@ def favourite(self):
             messagebox.showinfo("Error!",
                                 "No images found. Refresh and try again later",
                                 icon='warning')
-        copy(self.images[self.index], self.settings['fav-pics-folder'])
+        image = os.listdir(self.settings['pics-folder'])[self.index]
+        image_path = os.path.join(self.settings['pics-folder'], image)
```
Comment (abhsag24) [inline uiplib/gui/mainGui.py]: you can even do `image_path = self.images[self.index].image_path` see `uipImage.py`

Comment (SACHIN-13) [inline uiplib/gui/mainGui.py]: Aah yes! Will change it.

**Comment by abhsag24** [comment]
nice work @SACHIN-13  merging

**Pull request merged**